lib/util.js

```diff
@@ -367,6 +367,20 @@ const util = {
 
   signApp: (options = config.defaultOptions) => {
     console.log('signing ...')
+    if (config.targetOS === 'android') {
+      const apkDir = path.join(config.outputDir, 'apks')
+      const fullApkPath = path.join(apkDir, config.targetApkName)
+      const fullApkPathAligned = fullApkPath + '_aligned'
+      util.run('third_party/android_sdk/public/build-tools/29.0.2/zipalign', ['-f', '-p', 4, fullApkPath, fullApkPathAligned], options)
```

Comment:
So we're actually trying specifically to keep this kind of stuff out of js. Can you create a `sign_app` target in brave/build/android instead? Also it seems like the the `android_apk` template already does this through`package_apk`?

Reply:
```  # Creates a signed and aligned .apk.
  #
  # Variables
  #   apk_name: (optional) APK name (without .apk suffix). If provided, will
  #       be used to generate .info files later used by the supersize tool.
  #   assets_build_config: Path to android_apk .build_config containing merged
  #       asset information.
  #   deps: Specifies the dependencies of this target.
  #   dex_path: Path to classes.dex file to include (optional).
  #   packaged_resources_path: Path to .ap_ to use.
  #   output_apk_path: Output path for the generated .apk.
  #   native_lib_placeholders: List of placeholder filenames to add to the apk
  #     (optional).
  #   secondary_native_lib_placeholders: List of placeholder filenames to add to
  #     the apk for the secondary ABI (optional).
  #   native_libs: List of native libraries.
  #   native_libs_filearg: @FileArg() of additionally native libraries.
  #   secondary_abi_native_libs: (optional) List of native libraries for
  #     secondary ABI.
  #   secondary_abi_native_libs_filearg: (optional). @FileArg() of additional
  #     secondary ABI native libs.
  #   write_asset_list: Adds an extra file to the assets, which contains a list of
  #     all other asset files.
  #   keystore_path: Path to keystore to use for signing.
  #   keystore_name: Key alias to use.
  #   keystore_password: Keystore password.
  #   uncompress_shared_libraries: (optional, default false) Whether to store
  #     native libraries inside the APK uncompressed and page-aligned.```

Reply:
Using `package_apk` would be great, but for some reason they omitted private key password (`--key-pass`) parameter https://chromium.googlesource.com/chromium/src.git/+/refs/tags/79.0.3945.92/build/android/gyp/finalize_apk.py#27
We won't be able to use it without patching Chromium files.
So I'll make it through `sign_app` target as it doesn't require patching files.

Reply:
```
"--key-passwd",
invoker.keystore_password,
``` ?

Reply:
it is `--ks-pass`, we also need `--key-pass`. We need 2 passwords.